utils/mock-data-loader/tfm/mocks/users.js

```diff
@@ -170,4 +172,22 @@ module.exports = [
     firstName: 'ESTORE',
     lastName: 'ESTORE',
   },
+  {
```

Comment:
is `ESTORE` a valid team that we need to add everywhere? Even if not I think this should be
```
teams: [( id: 'ESTORE' }],
```
?